docs/src/docs/why-kitops.md

```diff
@@ -25,7 +25,7 @@ This problem is only getting worse and the stakes are rising each day as more an
 
 ## The Solution
 
-**The goal of KitOps is to keep a versioned package for your AI project in a repository you already use.**
+> **The goal of KitOps is to be a library of versioned packages for your AI project, in an OCI registry you already use.**
```

Comment:
just a quick FYI, you can use github flavored "alerts" for notes: https://vitepress.dev/guide/markdown#github-flavored-alerts it is supported!